### DISCUSSION TITLE
Add http health endpoint for argocd-notifications-controller and argo-rollouts-dashboard

### DISCUSSION THREAD

**Discussion by norbinor** [argoproj/argo-helm#2745] [Ideas]
Is it expected that you add http health endpoint implementation for argocd-notifications-controller and argo-rollouts-dashboard, so that we can use them in K8s readiness/liveness Probes health check. If it is expected, then when will it be possible?

**Comment by yu-croco** [comment]
Hi @norbinor 
I think it makes sense to provide probes. 🙋 
* If upstream ([argo-rollouts](https://github.com/argoproj/argo-rollouts/tree/master/manifests), [argo-cd](https://github.com/argoproj/argo-cd/tree/master/manifests)) provided, then we can provide as default.
* If not provided, we can provide as optional. e.g. [argo-cd applicationSet](https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/argocd-applicationset/deployment.yaml#L219-L228)) 

*We always welcome PRs. 😉
